src/write_gcm.c

```diff
@@ -2455,10 +2576,12 @@ typedef struct {
 
 typedef struct {
   _Bool pretty_print_json;
+  _Bool enable_metadata_agent;
```

Comment:
Naive question: why do we have enable_metadata_agent and metadata_agent_url defined both in this struct and in wg_configbuilder_t? Is there any reason we wouldn't want them defined in one place only?